cloudslang-compiler/src/test/java/io/cloudslang/lang/compiler/PreCompilerErrorsTest.java

```diff
@@ -139,6 +139,18 @@ public void testOperationWithWrongNameYMLExtension() throws Exception {
     }
 
     @Test
+    public void testOperationWithWrongNameWrongExtension() throws Exception {
```

Comment:
I think that it is better to have a test that is dedicated to only wrong extension, with correct name, so that you will know what you tested